client/src/components/Header.js

```diff
@@ -55,11 +55,19 @@ Header.propTypes = {
   setOpen: PropTypes.func,
 };
 const HeaderWrapper = styled.header`
-  background-color: #f27649;
+  background-color: var(--bg-color-highlight);
   width: 100%;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  text-align: center;
+
+  @media only screen and (max-width: 600px) {
+    h1 {
+      font-size: 1.8rem;
+      flex-grow: 2;
+    }
+  }
```

Comment:
I don't see how this is part of global styles :P
🔬

Reply:
I know... but it affected my global viewing ;) I will try not to mix it up in future PR